content/manual/os/dojo-tools.md

```diff
@@ -4060,6 +4170,19 @@ use |code %reset to invalidate this and generate a new code
 
 ---
 
+### `-code`
+
+Print out your web login code.
+
+#### Example
+
+```
+> -code
+lidlut-tabwed-pillex-ridrup
```

Comment:
shoud be `~lidlut-tabwed-pillex-ridrup`, no?

Reply:
yep, looks like it